src/goto-programs/goto_convert_class.h

```diff
@@ -588,6 +588,26 @@ class goto_convertt:public messaget
   bool get_string_constant(const exprt &expr, irep_idt &);
   exprt get_constant(const exprt &expr);
 
+  /// \brief Converts calls to the built in `enum_is_in_range` function to a
+  ///    test that the given enum value is in the valid range of values for that
+  ///    enum type.
+  ///
+  /// Note that the check for the range of values is done by creating a
+  /// disjunction comparing the expression with each possible valid value.
+  /// \param lhs The destination for the generated assignment.
```

Comment:
Nit picking: I believe the coding guidelines suggest to use a colon after the parameter's name.